### DISCUSSION TITLE
What's the difference between 4 large models? (large, large-v1, large-v2, large-v3)

### DISCUSSION THREAD

**Discussion by gety9** [SYSTRAN/faster-whisper#651] [Q&A]
Title. Which one is should use to transcribe speech to text?

**Comment by toanhuynhnguyen** [comment]
The best is large-v3

**Reply by tvone** [reply]
> The best is large-v3

larger-v3 will have more repeated words

**Reply by tvone** [reply]
I think large-v2 will give better results
